scripts/install_python_prod_dependencies_test.py

```diff
@@ -123,13 +123,28 @@ def __exit__(self, *args):
 
         self.cmd_token_list = []
         def mock_check_call(cmd_tokens, **unsued_kwargs):  # pylint: disable=unused-argument
-            self.cmd_token_list.append(cmd_tokens[2:])
+            if cmd_tokens and cmd_tokens[0].endswith('python'):
```

Comment:
Perhaps you should check for an os.path.sep before 'python'? Just in case some command is, like, abc-python or similar.

Reply:
done